### PR TITLE
UK TL: New Settings + items descriptions

### PR DESCRIPTION
**Pull request by tsunamistate**
Now that I'm on vacation, I get more time to TL, so expect more PRs
Reviewers: Ignore first commit
@SourceCodeDeleted please review when time permits

P.S. Merry Christmas and happy New Year to all DevilutuionX contributors!

### CODE REVIEW

File: Translations/uk.po [hunk 1]
```diff
@@ -6166,50 +6106,50 @@ msgstr "Підбір предметів видає звук підбору."
...
+msgstr "Роздільна Здатність"
 
 #: Source/options.cpp:661
 msgid ""
 "Affect the game's internal resolution and determine your view area. Note: "
 "This can differ from screen resolution, when Upscaling, Integer Scaling or "
 "Fit to Screen is used."
 msgstr ""
+"Діє на внутрішню роздільну здатність гри і визначає видиму область. "
+"Примітка: Настройка може відрізнятись від роздільної здатності монітору коли "
```
Comment (SourceCodeDeleted) [inline Translations/uk.po:6150]: Настройка is Russian  , налаштування is Ukrainian )

**Comment by SourceCodeDeleted** [comment]
Everything else seems good.

**Comment by tsunamistate** [comment]
@AJenbo @SourceCodeDeleted Fixed the review comment and rebased branch.
Good to merge as soon as checks pass

**Comment by AJenbo** [comment]
Looks like a healthy sized update :)

**Pull request merged**